### PR TITLE
[improve][doc] SEO for Security and Performance

### PR DESCRIPTION
**Pull request by zuobiao-zhou**
This PR do SEO for Security and Performance

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete` 

image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by zuobiao-zhou** [comment]
PTAL @Anonymitaet TIA.

File: docs/performance-pulsar-perf.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 id: performance-pulsar-perf
 title: Pulsar Perf
 sidebar_label: "Pulsar Perf"
+description: Learn ow to use Pulsar Perf for performance testing and HdrHistogram Plotter for visualizing test results.
```
Comment (Anonymitaet) [inline docs/performance-pulsar-perf.md:5]:
```suggestion
description: Learn how to use Pulsar Perf for performance testing and HdrHistogram Plotter for visualizing test results.
```

File: docs/security-extending.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 id: security-extending
 title: Extend Authentication and Authorization in Pulsar
 sidebar_label: "Extend Authentication and Authorization"
+description: Learn to use custom authentication and authorization mechanisms.
```
Comment (Anonymitaet) [inline docs/security-extending.md:5]:
```suggestion
description: Learn how to use custom authentication and authorization mechanisms.
```

**Comment by zuobiao-zhou** [comment]
fixed @Anonymitaet PTAL.

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**